migration/sql-files/077-assignee-and-label-empty-value.sql

```diff
@@ -0,0 +1,4 @@
+update work_items set fields=fields - 'system.assignees' where Fields->>'system.assignees' is null;
```

Comment:
could you put a short comment to explain the purpose of the SQL statements ?

Reply:
Done. cc60782a64b734c80d9a167ba0b1c1dd354e7824

Reply:
ok, thanks